src/components/pages/Onboarding/Onboarding.tsx

```diff
@@ -371,7 +378,7 @@ const mapStateToProps: MapStateToProps<ReduxStateProps, OwnProps, RahaState> = (
       state.authentication.isLoaded && state.authentication.isLoggedIn,
     deeplinkVideoToken,
     deeplinkInvitingMember,
-    isJointVideo: deeplinkInvitingMember ? true : false // TODO isJointVideo should be specified as part of the deeplink params
+    isJointVideo: deeplinkIsJointVideo ? true : false
```

Comment:
`!!deeplinkIsJointVideo` if need to force boolean

Reply:
ah yes 👍